upgrade/v100to101/upgrade.go

```diff
@@ -92,7 +92,8 @@ func doInstanceManagerUpgrade(namespace string, lhClient *lhclientset.Clientset)
 	}
 
 	for _, im := range imList.Items {
-		if im.Spec.Image != "" {
+		newInstanceManagerLabels := types.GetInstanceManagerLabels(im.Spec.NodeID, im.Spec.Image, im.Spec.Type)
```

Comment:
Can we split this part into another commit? It's specific for v1.0.0 to v1.0.1 upgrade.

Reply:
fixed